### PR TITLE
Form password customization

### PR DESCRIPTION
**Pull request by madassdev**
## Summary by CodeRabbit

- **New Features**
	- Enhanced forms display with additional properties such as description, theme, and SEO metadata.
	- Improved form aesthetics with theme-specific elements and logo customization.
	- Implemented conditional rendering for form descriptions for a cleaner user interface.

- **Enhancements**
	- Updated form components to support theming and color customization.
	- Added computed property to toggle visibility of form descriptions based on user preference.

### CODE REVIEW

File: client/components/open/forms/OpenCompleteForm.vue [hunk 1]
```diff
@@ -73,7 +73,7 @@
                    @submit="submitForm"
         >
           <template #submit-btn="{submitForm}">
-            <open-form-button :loading="loading" :theme="theme" :color="form.color" class="mt-2 px-8 mx-1"
+            <open-form-button :loading="loading" :theme="theme" color="green" class="mt-2 px-8 mx-1"
```
Comment (formsdev) [inline client/components/open/forms/OpenCompleteForm.vue:76]: Why it's fixed to "Green" ?

File: client/components/open/forms/OpenCompleteForm.vue [hunk 2]
```diff
@@ -162,6 +162,9 @@ export default {
     },
     isHideTitle () {
       return this.form.hide_title || (process.client && window.location.href.includes('hide_title=true'))
+    },
+    isHideDescription () {
+      return this.form.hide_description || (process.client && window.location.href.includes('hide_description=true'))
```
Comment (formsdev) [inline client/components/open/forms/OpenCompleteForm.vue:167]: you just create "isHideDescription" but not used anywhere

**Review by formsdev — CHANGES_REQUESTED** [review]

File: client/components/open/forms/OpenCompleteForm.vue (hunk 1 above)
Comment (madassdev) [reply client/components/open/forms/OpenCompleteForm.vue:76]: It was a mistake.
It's being resolved

**Review by formsdev — APPROVED** [review]
Good Work 👍

**Review by JhumanJ — APPROVED** [review]

**Review by JhumanJ — APPROVED** [review]

**Pull request merged**